schema/attributes.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$id": "https://opentelemetry.io/otelconfig/attributes.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Attributes",
```

Comment:
https://github.com/open-telemetry/opentelemetry-configuration/pull/6#issuecomment-1524153529

@jack-berg if we chose to keep the service name requirement, i agree that this should be moved into the resource file, i'm not sure if it will be based on @tsloughter's comment.

Reply:
Do you anticipate other uses for `Attributes`? I can't think of any other place in the sdk where they're used in configuration... Which is kind of surprising. 🤔

Reply:
🤔 hmmmm.... that's a good question. i guess not configurable attributes now that i think about it. I'll move them

Reply:
Moved attributes under resource.